src/wiki/templates/wiki/edit.html

```diff
@@ -17,11 +17,34 @@
 
   <div class="col-lg-8">
     <form method="POST" class="form-horizontal" id="article_edit_form">
-      <input type="hidden" name="save" value="1" />
-      <input type="hidden" name="preview" value="1" />
       {% with edit_form as form %}
         {% include "wiki/includes/editor.html" %}
       {% endwith %}
+
+      <div class="form-group form-actions">
+        <div class="col-lg-8">
+          <div class="col-lg-offset-2">
+            <button class="btn btn-default" type="submit" name="preview" value="1" id="id_preview"
+                    formaction="{% url 'wiki:preview' path=urlpath.path article_id=article.id %}"
+                    formtarget="previewWindow"
+                    >
+              <span class="fa fa-eye"></span>
+              {% trans "Preview" %}
+            </button>
+            <button class="btn btn-primary" type="submit" name="save" value="1" id="id_save">
+              <span class="fa fa-check"></span>
+              {% trans "Save changes" %}
+            </button>
+            {% if article|can_delete:user %}
+              <a href="{% url 'wiki:delete' path=urlpath.path article_id=article.id %}" class="btn btn-danger pull-right">
+                <span class="fa fa-trash-o"></span>
+                {% trans "Delete article" %}
+              </a>
+            {% endif %}
+          </div>
+        </div>
+      </div>
+
```

Comment:
This is from some years back - but I think the initial solution has been related to problems in getting the correct values to be sent together with the form.

But I can't find a proper source confirming that there are - for instance - browsers that don't send `value` together with the clicked submit button.

Furthermore, because the `Preview` button is the first button, that should also be the default trigger ENTER key codes.

So it should be fine :) Am glad to see JS be deleted :dancing_men: